src/main/frontend/idb.cljs

```diff
@@ -1,26 +1,24 @@
 (ns frontend.idb
-  (:require ["localforage" :as localforage]
-            [cljs-bean.core :as bean]
+  (:require [cljs-bean.core :as bean]
             [goog.object :as gobj]
             [promesa.core :as p]
             [clojure.string :as string]
             [frontend.config :as config]
             [frontend.util :as util]
-            [frontend.storage :as storage]))
+            [frontend.storage :as storage]
+            ["/frontend/idbkv" :as idb-keyval :refer [Store]]))
+
 
 ;; offline db
-(def store-name "dbs")
-(.config localforage
-         (bean/->js
-          {:name "logseq-datascript"
-           :version 1.0
-           :storeName store-name}))
 
-(defonce localforage-instance (.createInstance localforage store-name))
+;; To maintain backward compatibility
+
+
+(defonce store (Store. "localforage" "keyvaluepairs" 2))
```

Comment:
Why is it called "localforage"?

Reply:
@gyk  might be from https://github.com/localForage/localForage

Reply:
@defclass Yes, but it no longer depends on localForage. Does using the old name maintain backward compatibility?

Reply:
yep, the old db namespace is "localforage", I should rename it earlier :)

Reply:
ahh,  I don’t know the program is no longer dependent on it  :  )